README.md

```diff
@@ -44,15 +44,24 @@ This list contains tools, books, articles, blogs, courses and everything related
 ## Apps
 
 - [RTranslator](https://github.com/niedev/RTranslator) - The first simultaneous translator app for Android based on Google's API.
+
 - [Crow Translate](https://github.com/crow-translate/crow-translate) - A simple and lightweight translator for Linux and Windows that allows to translate and speak text using Google, Yandex and Bing translate API.
+
 - [Copy Translator](https://github.com/CopyTranslator/CopyTranslator) - An app for Windows, Mac and Linux that automatically translate and adapt a text when copied in the clipboard.
+
 - [Electronjs i18n](https://www.electronjs.org/apps/i18n-manager) - I18n manager for Windows.
+
 - [OmegaT](https://omegat.org/) - Free translation memory application that works on Windows, macOS and Linux.
+
 - [Argos Translate](https://github.com/argosopentech/argos-translate) - Completely open source offline translation app written in Python and based on OpenNMT. It can also be used as a Python library.
+
 - [LibreTranslate](https://github.com/uav4geo/LibreTranslate) - 100% Self-Hosted web app to translate text, based on Argos Translate.
 
+  
+
```

Comment:
Please don't add free/blank/empty space. Redundant action here

Reply:
Sorry, I will fix this issue. Thank you.

Reply:
I have made the requested changes to the pull request. Thank you for the review.